TKeazirian.HTTPServer/SocketHandler.cs

```diff
@@ -15,5 +15,6 @@ public static void CloseSocketConnection(Socket handler)
     {
         handler.Shutdown(SocketShutdown.Both);
         handler.Close();
+        handler.Dispose();
```

Comment:
> added dispose and closing socket calls

Is a `handler` actually a socket? I think I am confused here.

Reply:
Yes, the `handler` is the socket.  I can call it `socket` to be more clear.  It is originally `listener` when it is first instantiated (in `CreateSocketListener`) and then I rename it `handler`.  But `socket` will be more clear, I think?

Reply:
If its actually a socket, I'd call it a socket.  It HTTP definitions, a `handler` would handle an event.